en/includes/references/grant-types.md

```diff
@@ -52,29 +114,87 @@ The following diagram shows how the refresh token flow works.
 1. The client application requests user information from the resource server by providing the access token.
 2. As the access token is expired, the resource server returns an error message.
 3. By sending the refresh token, the client application requests a new access token from {{ product_name }}.
+
+    === "Request format (/token)"
+
+        ```bash
+        curl -k {{base_url}}/oauth2/token \
+        --header "Content-Type: application/x-www-form-urlencoded"
+        --header "Authorization: Basic <Base64Encoded(CLIENT_ID:CLIENT_SECRET)>" \
+        --data-urlencode "grant_type=refresh_token" \
+        --data-urlencode "refresh_token=<REFRESH_TOKEN>"
+        ```
+    
+    === "Sample request (/token)"
+
+        ```bash
+        curl -k {{base_url_example}}/oauth2/token \
+        --header "Content-Type: application/x-www-form-urlencoded"
+        --header "Authorization: Basic RWkwV2Y5YnpmTXE0UTBsZndTdlRQamU4a2NFYTpIRvvyUzJIUjlrZE9YMjBXTG9JNmY1eE1wdUlBamdKeG5aUVVUMV9lNTJnYQ==" \
+        --data-urlencode "grant_type=refresh_token" \
+        --data-urlencode "refresh_token=96a6dsd97-0120-3bec-86be-21b58f600a07"
+        ```
+
 4. {{ product_name }} sends a new access token and a new refresh token.
 
+    ```json
+    {
+    "access_token":"b9ed0658-f187-3d39-a4f1-6d42522e1eee",
+    "refresh_token":"3426ff78-62a5-32fa-be6e-74ab69d4cbf4",
+    "token_type":"Bearer",
+    "expires_in":3600
+    }
+    ```
 
 ## Client credentials grant
 
-The client credentials flow provides a secure way for client applications to obtain an access token without user authentication. This can be useful in scenarios where the client application needs to access its own resources, such as data storage or APIs, but does not require access to user data.
+The client credentials flow provides a secure way for client applications to obtain an access token without user authentication. This can be useful in scenarios where the client application needs to access its own resources, such as data storage or APIs, without requiring user authentication.
```

Comment:
Let's say that this grant type is commonly used for machine machine communication